x-pack/test/functional/apps/aiops/change_point_detection.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const elasticChart = getService('elasticChart');
+  const esArchiver = getService('esArchiver');
+  const aiops = getService('aiops');
+
+  // aiops lives in the ML UI so we need some related services.
+  const ml = getService('ml');
+
+  describe('change point detection', async function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/ecommerce');
+      await ml.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.testResources.deleteIndexPatternByTitle('ft_ecommerce');
+    });
+
+    it(`loads the change point detection page`, async () => {
+      // Start navigation from the base of the ML app.
+      await ml.navigation.navigateToMl();
+      await elasticChart.setNewChartUiDebugFlag(true);
```

Comment:
What's this flag for? Is it needed?

Reply:
added chart state assertion in https://github.com/elastic/kibana/pull/158164/commits/d6d43232421eb24e3256c6d6710837d01ea6d117